Models/ViewModel.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Models;
+
+class ViewModel
+{
+    public function __construct()
+    {
+
+    }
+
+    public function renderIndex()
+    {
+        ob_start();
+        include_once("Views/header.phtml");

```

Comment:
Please use template file rendering as i sent you in my project. This one looks really awful. I can smell a shit around me. But maybe i have to take a shower ...